identity/app/controllers/editprofile/EditProfileFormHandling.scala

```diff
@@ -89,7 +89,9 @@ trait EditProfileFormHandling extends EditProfileControllerComponents {
                   logger.error(s"Failed to process ${page.id} form submission for user ${userDO.getId}: $idapiErrors")
                   profileFormsView(page, boundProfileForms.withErrors(idapiErrors), userDO)
 
-                case Right(updatedUser) => profileFormsView(page, boundProfileForms.bindForms(updatedUser), updatedUser)
+                case Right(updatedUser) =>
+                  val userChangedEmail: Option[String] = if (formData.toUserUpdateDTO(userDO).primaryEmailAddress.getOrElse(userDO.primaryEmailAddress) == userDO.primaryEmailAddress) None else formData.toUserUpdateDTO(userDO).primaryEmailAddress
```

Comment:
I think this should be a filter or filterNot on `formData.toUserUpdateDTO(userDO).primaryEmailAddress`

Reply:
Alternatively would using `updatedUser` be simpler?

Reply:
Yep! seems obvious now, thanks